environs/networking.go

```diff
@@ -21,6 +21,7 @@ var SupportsNetworking = supportsNetworking
 // to get information about the default space.
 var DefaultSpaceInfo *network.SpaceInfo
 
+// TODO (manadart 2019-07-03): Move this to core.
```

Comment:
Do it now?

Reply:
It is used in a bunch of places that would add chafe to this patch.